examples/eventlisteners/eventlistener-podTemplate.yaml

```diff
@@ -0,0 +1,17 @@
+---
+apiVersion: triggers.tekton.dev/v1alpha1
+kind: EventListener
+metadata:
+  name: listener-podTemplate
+spec:
+  serviceAccountName: tekton-triggers-example-sa
```

Comment:
Thanks for adding examples, can you also update the docs to mention the new fields?

Reply:
I added `replicas` and `podTemplate.nodeSelector` to docs. :)